utils/gyb_syntax_support/Node.py

```diff
@@ -58,7 +58,7 @@ def is_syntax_collection(self):
 
     def requires_validation(self):
```

Comment:
Nitpick: This function can be deleted

Reply:
It is still used here: https://github.com/apple/swift/blob/6fc4cef671947d1d2482951853f7f82854473d91/include/swift/Syntax/SyntaxNodes.h.gyb#L75